azavea/echo-locator
# Add tooltips to profile page

## Overview

Creates tooltips on the profile page for the following importance preferences: Commute Time, School Quality, and Public Safety. Aside from small design changes detailed in notes, this was intended to replicate the tooltips on the demo site. Translations for the tooltips were found in the translations file, so those have been added as well.

### Demo
Tooltips should display to the right on icon hover and be properly translated.
![Screen Shot 2022-01-27 at 4 12 29 PM](https://user-images.githubusercontent.com/36374797/151444420-58d27094-3e87-4054-a3b9-55f4818e3603.png)
![Screen Shot 2022-01-27 at 4 12 20 PM](https://user-images.githubusercontent.com/36374797/151444406-4754f891-6720-411a-b167-17a1cb6b0a59.png)

### Notes

- I made these icons centered between the label and the options select input. However, because each label text is of slightly different length they are not vertically aligned. I left it because it matches the demo site more closely, but can set to just `margin-left:auto` so that they are not reliant on the length of the label and will all be vertically aligned. EDITED: icons now sit on the right of select.

- I moved the tooltips to displaying on the right of the icon instead of the top so that a user can more easily hover up to next icon, an action previously blocked by the appearing tooltip.

-  I made the classNames for these tooltips the same as the tooltips elsewhere in the app to keep styling consistent.


## Testing Instructions

 * Run ./scripts/server
 * Login and navigate to '/profile'
 * Confirm tooltips appear and match the demo site
 * Select different translations and make sure each tooltip appears and has proper translations


Resolves #404
